src/types/mapping.rs

```diff
@@ -102,17 +108,52 @@ impl PyMapping {
                 .from_owned_ptr_or_err(ffi::PyMapping_Items(self.as_ptr()))
         }
     }
+
+    /// Register a pyclass as a subclass of `collections.abc.Mapping` (from the Python standard library).
+    /// This is required for a class to be downcastable from `PyAny` to `PyMapping`.
```

Comment:
Might be nice to add "this is equivalent to `collections.abc.Mapping.register(T)`,  like we have for other docs with python equivalent functionality?